docusaurus.config.js

```diff
@@ -168,7 +168,7 @@ module.exports = {
           ],
         },
       ],
-      copyright: `Copyright © ${new Date().getFullYear()} Tolgee`,
+      copyright: `© 2021-${new Date().getFullYear()} Tolgee s.r.o. All rights reserved`,
```

Comment:
I think, that he only wanted us to make it a link to homepage.

Reply:
Ah, yeah I've misunderstood